waspc/data/Generator/templates/server/src/jobs/core/pgBoss/pgBossJob.js

```diff
@@ -55,7 +55,7 @@ class PgBossJob extends Job {
    * @param {object} jobArgs - The job arguments supplied by the user for their perform callback.
    * @param {object} jobOptions - pg-boss specific options for `boss.send()`, which can override their defaultJobOptions.
    */
-  async submit(jobArgs, jobOptions) {
+  async submit(jobArgs, jobOptions = {}) {
```

Comment:
@shayneczyzewski We're using this as an optional parameter and TS complains if we don't specify a default value.